faqs/galaxy/histories_import.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Import an history
+title: Importing history
```

Comment:
I would make it either "Importing histori**es**" (plural), or "Importing **a** history"

Reply:
Yeah to make it consistent with the other questions on the GTN history section I will use the second one. Thanks